### ISSUE TITLE
Introducing comments to the CV entries

### ISSUE THREAD

**Issue by julianu** [HUPO-PSI/mzQC#83]
To better understand the interpretation of the CV entries, or rather the values in the mzQC file itself, I would like to introduce comments to the obo files.
This will be in the next pull request (containing the entries from issue #52 ) and needs to be done for all metrics. Help is very welcome for this.

**Comment by mwalzer** [comment]
:+1: 
So I guess this will be a thing of ongoing refinement: so should we suggest comments/changes in the respective original cv issue, new issue, PR or elsewhere?
Interesting though is to have an open thread for each cv term. Worry with this is it might get too long and confusing.

**Comment by mwalzer** [comment]
wiki might be another place, but it's bad for discussions.

**Comment by mwalzer** [comment]
This might actually help us some 
1. present our terms as clear as possible for a #141 PR
2. our terms being mixed in with general MS terms, PEFF terms, etc. (after #141) it will help avoid misinterpretation in the future even more

gets a priority :+1: from me

**Comment by bittremieux** [comment]
[As discussed recently](https://github.com/HUPO-PSI/mzQC/blob/main/meeting_notes/2022/20220112_telco.md), we don't want to overdo it and introduce unnecessary comments. However, we do want to include comments when they can help to clarify how to interpret metrics.

The first batch of terms proposed for inclusion into the PSI-MS-CV illustrates this perfectly. Although this remains an ongoing point of attention as we merge more terms into the PSI-MS-CV, I'll close this issue for now.
